asmcli/asmcli.sh

```diff
@@ -327,9 +336,12 @@ configure_kubectl(){
   local ADDR
   ADDR="$(kubectl config view --minify=true -ojson | \
     jq .clusters[0].cluster.server -r)"
+  local DEST
+  DEST="${ADDR:8:${#ADDR}}"
+  DEST="${DEST%:*}"
 
   local RETVAL; RETVAL=0;
-  run_command nc -zvw 10 "${ADDR:8:${#ADDR}}" 443 || RETVAL=$?
+  run_command nc -zvw 10 "${DEST}" 443 || RETVAL=$?
```

Comment:
nit: I don't think ADDR is used anymore so you can just keep using that variable instead of adding a new one

Reply:
done